controls/V-71943.rb

```diff
@@ -80,7 +83,11 @@
     }
     its('lines') { should match_pam_rule('auth [default=die]|required pam_faillock.so').all_with_integer_arg('deny', '<=', unsuccessful_attempts) }
     its('lines') { should match_pam_rule('auth [default=die]|required pam_faillock.so').all_with_integer_arg('fail_interval', '<=', fail_interval) }
-    its('lines') { should match_pam_rule('auth [default=die]|required pam_faillock.so').all_with_args('unlock_time=(604800|0|never)') }
+    its('lines') {
+      should match_pam_rule('auth [default=die]|required pam_faillock.so').all_with_args('unlock_time=(0|never)').or \
+            (match_pam_rule('auth [default=die]|required pam_faillock.so').all_with_integer_arg('unlock_time', '<=', 604800).and \
```

Comment:
This is reversed. The prose indicates that `604800` is the minimum value that can be used.

Reply:
Ok, I realized what I was seeing. The DISA text says that it has to be `<=` to the number (which is what your check has in place). However, your comments, indicate that `604800` is the minimum and are out of sync with the DISA prose so should be corrected.